Setup.pm

```diff
@@ -35,21 +43,43 @@ sub update_win32_shell() {
     return;
 }
 
-sub create_shortcut {
-    my $start_menu_path = Win32::GetFolderPath(Win32::CSIDL_STARTMENU());
-    my $start_menu_base = catfile($start_menu_path, 'ActiveState');
+sub desktop_dir_path() {
```

Comment:
The trailing `()` on a function definition is counterproductive here and elsewhere.